docs/src/gui/gmoccapy.adoc

```diff
@@ -1142,14 +1142,20 @@ image::images/gmoccapy_keyboard_edit_mode.png["Onboard keyboard",align="left"]
 
 .On Touch Off
 
-This gives the option whether to show the preview tab or the offset page tab when
- you enter the touch off mode by clicking the corresponding bottom button.
+This gives the option whether to show the preview tab or the offset page tab when you enter the touch off mode by clicking the corresponding bottom button.
 
 * _show preview_
 * _show offsets_
 
-.DRO Options
+.Show Auxiliary Screen (Is this still valid?)
+Clicking this button will open an additional window.
+This button is only sensitive if a file named 'Gmoccapy 3.glade' is in your configuration folder.
+You can build the Aux screen using Glade.
+
+[WARNING]
+The main window of the auxiliary screen must be named 'window2'.
```

Comment:
This is obsolete therefore I removed it.

Reply:
I pulled from my branch prior to the rebase - I should have it :)

Reply:
@hansu  - now removed, I had understood that you'd have done so already.

Reply:
Ah ok, now I understand your comment :).

I meant I removed it on master.